azure-cli-diff-tool/setup.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+
+# -----------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -----------------------------------------------------------------------------
+
+"""Azure Command Diff Tools package that can be installed using setuptools"""
+
+from setuptools import setup, find_packages
+
+
+with open('README.rst', 'r', encoding='utf-8') as f:
+    README = f.read()
+with open('HISTORY.rst', 'r', encoding='utf-8') as f:
+    HISTORY = f.read()
+
+setup(name="azure-cli-diff-tool",
+      version='0.0.1',
```

Comment:
According to the following rules, the first extension version should be 1.0.0b1
```
How to determine if the extension should be stable or preview?  
If the extension contains preview api-version (like "2020-01-01-preview"), it will be preview.  
If the extension is the first release, it will be preview (1.0.0b1). 
Exception: the owner of the extension can decide if the extension will be stable or preview if they think the rules above are not reasonable. 
```
![image](https://github.com/Azure/azure-cli-extensions/assets/18628534/394d9b48-9676-40dd-942a-9ed16c1da669)

Reply:
The az extension module for users which depends on this tool will be started releasing from `1.0.0b1`. 
For this developers tool, I would recommend its version starts from a more minor one which is `0.0.1`

Reply:
[aaz-dev](https://pypi.org/project/aaz-dev/) starts from `1.0.0`, maybe we can align the versioning with it.